### ISSUE TITLE
Cannot set useAutomationExtension option for chrome

### ISSUE THREAD

**Issue by kullmanp** [arquillian/arquillian-extension-drone#114]
##### Issue Overview

I would like to set the experimental option "useAutomationExtension" to false. 

##### Expected Behaviour
It should be possible like this:

```
  <extension qualifier="webdriver">
    <property name="chromeExperimentalOption">
    {
      "useAutomationExtension": "false"
    }
    </property>
  </extension>

```

##### Current Behaviour
I get an exception:

```
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at path $
	at com.google.gson.Gson.fromJson(Gson.java:899)
	at com.google.gson.Gson.fromJson(Gson.java:952)
	at org.jboss.arquillian.drone.webdriver.factory.CapabilitiesOptionsMapper.handleDictionaries(CapabilitiesOptionsMapper.java:108)
	at org.jboss.arquillian.drone.webdriver.factory.CapabilitiesOptionsMapper.mapCapabilities(CapabilitiesOptionsMapper.java:86)
	at org.jboss.arquillian.drone.webdriver.factory.ChromeDriverFactory.setChromeOptions(ChromeDriverFactory.java:123)
	at org.jboss.arquillian.drone.webdriver.factory.ChromeDriverFactory.getCapabilities(ChromeDriverFactory.java:114)
	at org.jboss.arquillian.drone.webdriver.factory.ChromeDriverFactory.createInstance(ChromeDriverFactory.java:79)
	at org.jboss.arquillian.drone.webdriver.factory.ChromeDriverFactory.createInstance(ChromeDriverFactory.java:41)
	at org.jboss.arquillian.drone.webdriver.factory.WebDriverFactory.createInstance(WebDriverFactory.java:127)
	at org.jboss.arquillian.drone.webdriver.factory.WebDriverFactory.createInstance(WebDriverFactory.java:38)
	at org.jboss.arquillian.drone.impl.DroneConfigurator$1.createInstance(DroneConfigurator.java:112)
	at org.jboss.arquillian.drone.impl.CachingCallableImpl.call(CachingCallableImpl.java:44)
	at org.jboss.arquillian.core.impl.threading.ThreadedExecutorService$ContextualCallable.call(ThreadedExecutorService.java:89)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at path $
	at com.google.gson.internal.bind.JsonTreeReader.expect(JsonTreeReader.java:162)
	at com.google.gson.internal.bind.JsonTreeReader.beginObject(JsonTreeReader.java:87)
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:183)
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145)
	at com.google.gson.Gson.fromJson(Gson.java:887)
	... 16 more
Starting ChromeDriver 2.32.498550 (9dec58e66c31bcc53a9ce3c7226f0c1c5810S9tarting ChromeDriver 2.32.498550 (9dec58e66c31bcc53a9ce3c7226f0c1c5810906a) on port 47233

```

##### Steps To Reproduce
Just add this option.

##### Additional Information
Debugging the drone code I can see that in the `org.jboss.arquillian.drone.webdriver.factory.CapabilitiesOptionsMapper` the arguments to 
setExperimentalOpion are expected to be maps. Primitives should also be handled in some way.

**Comment by MatousJobanek** [comment]
Hi @kullmanp,
thanks a lot for reporting. Good catch - small problem in the parser. 
The fix should be simple and quick.

**Comment by MatousJobanek** [comment]
Yesterday was released Drone 2.4.4 with the fix for this issue.
